src/Workspaces/Core/Portable/Recommendations/AbstractRecommendationServiceRunner.cs

```diff
@@ -161,6 +147,39 @@ private ImmutableArray<ITypeSymbol> GetTypeSymbols(ImmutableArray<ISymbol> candi
             return builder.ToImmutableAndFree().Distinct();
         }
 
+        private bool TryGetMatchingParameterTypeForArgument(IMethodSymbol method, string argumentName, int ordinalInInvocation, out ITypeSymbol parameterType)
+        {
+            if (!string.IsNullOrEmpty(argumentName))
+            {
+                parameterType = method.Parameters.FirstOrDefault(p => p.Name == argumentName)?.Type;
```

Comment:
note: VB is case insensitive.

Reply:
So this is broken too? http://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic.Workspaces/Extensions/ArgumentSyntaxExtensions.vb,51

Reply:
Definitely looks like it!

Reply:
I'm now getting the StringComparer from the SyntaxFacts.